shared/util/forward-logs.js.flow

```diff
@@ -0,0 +1,7 @@
+// @flow
+export const setupSource = () => {}
```

Comment:
you're either a source of logs (browserwindows etc) or a target of logs (main thread)
we also have access to the underlying console.logs